src/kvstore/raftex/Host.cpp

```diff
@@ -190,6 +190,7 @@ void Host::appendLogsInternal(folly::EventBase* eb,
             {
                 std::lock_guard<std::mutex> g(self->lock_);
                 self->setResponse(r);
+                self->lastLogIdSent_ = self->logIdToSend_;
```

Comment:
when send log failed, why update the last sent log id?

Reply:
That would cause leader trying to send snapshot and scanning rocskdb.